harmonica/tests/test_transformations.py

```diff
@@ -7,75 +7,135 @@
 """
 Test functions for regular grid transformations
 """
-import os
-from pathlib import Path
-
 import numpy as np
 import pytest
-import xarray as xr
+import verde as vd
 import xrft
 
+from .. import point_gravity
 from ..transformations import derivative_upward
 from .utils import root_mean_square_error
 
-MODULE_DIR = Path(os.path.dirname(__file__))
-TEST_DATA_DIR = MODULE_DIR / "data"
+
+@pytest.fixture(name="sample_sources")
+def fixture_sample_sources():
+    """
+    Define a pair of sample point sources used to build tests
+    """
+    points = [
+        [-50e3, 50e3],
+        [-50e3, 50e3],
+        [-20e3, -20e3],
```

Comment:
might be good to have one shallow source and one deep source?

Reply:
Sure! Well thought!